src/tests/Microsoft.PowerFx.Core.Tests/PrimitiveValueConversionsTests.cs

```diff
@@ -37,21 +39,23 @@ public void TestAll(Type dotnetType, Type fxType)
             else
             {
                 Assert.True(result);
-
                 Assert.True(actualFxType.GetType() == fxType);
 
                 var value = GetValue(dotnetType);
                 var fxValue = PrimitiveValueConversions.Marshal(value, dotnetType);
                 Assert.Equal(fxType, fxValue.Type.GetType());
 
                 var expr = actualFxType.DefaultExpressionValue();
-                var engine = new Engine(new PowerFxConfig());
+
+                SymbolTable symbol = new SymbolTable();
+                symbol.AddFunctions(BuiltinFunctionsCore._featureGateFunctions);
+                var engine = new Engine(new PowerFxConfig() { SymbolTable = symbol });
```

Comment:
nit: can we encapsulate this in a helper:
GetEngineWithFeatureGatedFunctions();

Reply:
yes, update the code